training/04_editing_eml.Rmd

```diff
@@ -1,14 +1,13 @@
 # Editing EML
 
-This chapter is a practical tutorial for using R to read, edit, write, and validate EML documents. Much of the information here can also be found in the vignettes for the R packages used in this section (e.g. the <a href = 'https://github.com/ropensci/EML/blob/master/vignettes/creating-EML.Rmd' target='_blank'>EML package</a>).
+This chapter is a practical tutorial for using R to read, edit, write, and validate EML documents. Much of the information here can also be found in the vignettes for the R packages used in this section (e.g. the <a href = 'https://github.com/ropensci/EML/blob/master/vignettes/creating-EML.Rmd' target='_blank'>EML package</a>). 
+
+Most of the functions you will see in this chapter will use the `arcticdatautils` package.
```

Comment:
Is this true? I guess maybe they are all EML wrappers in arcticdatautils but some are definitely from EML itself

Reply:
Right - I'll update the language to reflect the use of `arcticdatautils` and `EML` . Adding that there I was mainly hoping to help orient the users to the packages we are using with out having to explicitly write arcticdatautils:: and EML:: each time